pegasus/sites.v3/hourofcode.com/views/map.haml

```diff
@@ -1,16 +1,32 @@
+%link{rel: "stylesheet", type: "text/css", href: "/css/map.css"}
+
 -total_count = DB[:forms].where(kind:'HocSignup2015').count
 -if @company
-  %h2= "<a href='#{resolve_url('/events')}'>#{hoc_s(:map_header).gsub('#', format_integer_with_commas(total_count))}</a>" + ", " + "#{hoc_s(:map_header_company).gsub('#', format_integer_with_commas(company_count(@company))).gsub('company', hoc_s("company_" + @company))}"
+  -company_name = hoc_s("company_" + @company)
+  -company_total = format_integer_with_commas(company_count(@company))
+  %a.float-left{href: resolve_url('/events/all')}
+    %h2.mapheader
+      = hoc_s(:map_header).gsub('#', format_integer_with_commas(total_count)) + ","  
+      = hoc_s(:map_header_company).gsub('#', company_total).gsub('company', company_name)
+  %a{href: resolve_url("/#join")}
+    %button.signup-button
+      = hoc_s(:signup_your_event)
+-elsif @country != 'us'
```

Comment:
Typically I like to keep the US side really simple, and I don't update EN unless Rox/Laura have the same request.